doc/source/tools/postprocessing/postprocessing.rst

```diff
@@ -0,0 +1,74 @@
+
+------------------------------
+Importing lethe_pyvista_tools
+------------------------------
+
+There are two ways to have access to ``lethe_pyvista_tools``: you can install the module and have it on your machine just like ``os``, ``sys``, or other PyPi modules; or, you can use it without installing. Here, we explain how to do both.
+
+
+Installing
+~~~~~~~~~~
+
+Installing ``lethe_pyvista_tools`` is very simple. In the `contrib/postprocessing <https://github.com/lethe-cfd/lethe/tree/master/contrib/postprocessing>`_ of your Lethe version you will find all source files, including ``setup.py`` and ``requirements.txt``. Those two files take care of configuring your installation and assuring that everything is set. To install it, navigate to `contrib/postprocessing <https://github.com/lethe-cfd/lethe/tree/master/contrib/postprocessing>`_ on your terminal and run:
+
+
+.. code-block::
+
+    pip3 install .
+
+or
+
+.. code-block::
+
+    pip install .
+
+.. note::
+
+    If you do not have ``pip`` on your machine, run ``sudo apt install python3-pip``.
+
+.. tip::
+
+    Uninstalling ``lethe_pyvista_tools`` is exactly like doing so to any other library: ``pip3 uninstall lethe_pyvista_tools`` or ``pip3 uninstall lethe_pyvista_tools``.
+
+Importing without installing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this case, we use the `sys <https://docs.python.org/3/library/sys.html>`_ module to import it directly from Lethe's directory:
+
+.. code-block::
+
+  import sys
+  path_to_module = '$LETHE_PATH/contrib/postprocessing/'
+  sys.path.append(path_to_module)
+  from lethe_pyvista_tools import *
+  import matplotlib.pyplot as plt
+
+where `sys.path <https://docs.python.org/3/library/sys.html#sys.path:~:text=in%20version%203.10.-,sys.path%C2%B6,-A%20list%20of>`_ is a list of strings that specifies the search path for modules. However, one can simply copy the `lethe_pyvista_tools.py <https://github.com/lethe-cfd/lethe/tree/master/contrib/postprocessing>`_ file to the same folder as the Python post-processing script is and import it, such as:
+
+.. code-block::
+
+  from lethe_pyvista_tools import *
+
+One third and very convenient way to always import the module without copying it or even adding the ``sys.path.append(path_to_module)`` is permanently adding the path to the module (``/contrib/postprocessing/``) to your `PYTHONPATH <https://docs.python.org/3/library/sys_path_init.html#:~:text=The%20PYTHONPATH%20environment%20variable%20is,all%20installed%20Python%20versions%2Fenvironments.>`_.
+
+The ``*`` means that we want to import all members of lethe_pyvista_tools.

```

Comment:
This should be moved right after the last code block, no? Before the "One third and very convenient...)

```suggestion
The ``*`` means that we want to import all members of lethe_pyvista_tools.

```